[2022.10.20] Feature/projects/:projectId GET api 구현

rktnsinger opened the pull request:
## 작업 사항
- /projects/:projectId GET 요청에 대한 api를 작성하였습니다
- 요청시, 인증된 사용자인지 확인 후 파라미터로 전달된 projectId와 매치되는 project의 정보를 응답합니다
## 잘 봐주세요
- secret key 응답 과정

## PR 전 확인사항
* [x] 가장 최신 브랜치를 pull했습니다.
* [x] base 브랜치명을 확인했습니다.
* [x] 코드 컨벤션을 모두 지켰습니다.
* [x] 적절한 라벨이 있습니다.
* [x] assignee가 있습니다.

## 비고
- 팀 컨벤션과 다르게 작성된 폴더명을 수정하였습니다
- 클라이언트 측 인증상태 지속에 쿠키를 사용하기로 한 논의 결과에 따라 httpOnly 설정을 변경하였습니다

Commit:
chore: 팀 컨벤션과 다르게 작성된 폴더명 변경 및 httpOnly cookie 설정 변경
- config, middlewares로 변경
- fe에서 쿠키 사용하기 위하여 httponly 설정 변경

Commit:
feat: /projects/:projectId GET api 구현 완료

Commit:
chore: package.json 변경

Yanguk reviewed (commented):
고생하셨어요! 아주 깔끔하게 잘 해주신 것 같아요! 코멘트 확인 해주세요~ 👍

ssong-yi reviewed (approved):
형우님 고생많으셨어요~

항상 꼼꼼히 작업해주셔서 감사합니다!

Commit:
refactor: 코드리뷰 반영
- method chaining 줄 나누어 작성
- 입력값 유효성에 대하여 발생하는 에러는 최종 에러핸들러에서 로그하기위해 에러만 발생시키고 넘김
- getProject 응답 로직 변경, 최종 응답값은 동일

Yanguk reviewed (approved):
고생하셨습니다~ 👍 👍
